src/configuration/builder.rs

```diff
@@ -49,6 +49,10 @@ impl ConfigurationBuilder {
     self.insert("lineWidth", (value as i32).into())
   }
 
+  pub fn indent_width(&mut self, value: u32) -> &mut Self {
+    self.insert("indentWidth", (value as i32).into())
+  }
```

Comment:
Have you tested this out? I don't think it will work. I never added indent width to markdown because markdown is very specific about how indentation works.

Reply:
I will add a test to verify if it works correctly. Thank you for bringing this to my attention.

Reply:
@dsherret You’re right. I tried testing it after the modification, but it failed. Do you have any suggestions that could help resolve this issue?